File: Basic Coding Problems/01 Fundamentals/004 Rectangle Perimeter/Java/RectanglePerimeter.java
import java.util.Scanner;

public class RectanglePerimeter {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the length of the rectangle: ");
        int length = scanner.nextInt();

        System.out.print("Enter the breadth of the rectangle: ");
        int breadth = scanner.nextInt();

        int perimeter = 2 * (length + breadth);

        System.out.println("The perimeter of the rectangle is: " + perimeter);

        scanner.close();
    }
}

File: Basic Coding Problems/01 Fundamentals/004 Rectangle Perimeter/C++/RectanglePerimeter.cpp
#include <iostream>

using namespace std;

int main() {
    int rectangleLength, rectangleBreadth, rectanglePerimeter;

    cout << "Enter the length of the rectangle: ";
    cin >> rectangleLength;

    cout << "Enter the breadth of the rectangle: ";
    cin >> rectangleBreadth;

    rectanglePerimeter = 2 * (rectangleLength + rectangleBreadth);

    cout << "The perimeter of the rectangle is: " << rectanglePerimeter << endl;

    return 0;
}